api/app/Jobs/HandlePushWebhookJob.php

```diff
@@ -28,12 +30,24 @@ public function handle()
     $payload = $this->webhookCall->payload();
     $payload = json_decode($payload['payload']);
     $commit = $payload->commits[count($payload->commits) - 1];
+    $message = [
+      'type' => 'commit',
+      'name' => $payload->sender->login,
+      'avatar' => $payload->sender->avatar_url,
+      'repository' => $payload->repository->full_name,
+      'commit_url' => $commit->url
+    ];
     if ($commit->committer->username !== 'web-flow') {
-      $project = Project::with('members.user')->where('repository', $payload->repository->name)->first();
+      $project = Project::with(['members.user', 'messages'])->where('repository', $payload->repository->full_name)->first();
       if ($project) {
```

Comment:
Try searching about Laravel scopes, we can shorten the `where` part of this line

[https://laravel.com/docs/9.x/eloquent#local-scopes](https://laravel.com/docs/9.x/eloquent#local-scopes)